prime-dotnet-webapi/Startup.cs

```diff
@@ -72,6 +72,7 @@ public void ConfigureServices(IServiceCollection services)
             services.AddScoped<IEnrolmentCertificateService, EnrolmentCertificateService>();
             services.AddScoped<IGisService, GisService>();
             services.AddScoped<IHealthAuthorityService, HealthAuthorityService>();
+            services.AddScoped<IClientLogService, ClientLogService>();

```

Comment:
In a PR last week I organized service declarations in alphabetical order, please adjust

Reply:
Yeah forgot to shift after name change